### ISSUE TITLE
Sound Environments don't work

### ISSUE THREAD

**Issue by TRBlount** [scp-fs2open/fs2open.github.com#1409]
While working on the Sound Environment Editor for qtFRED today, @AxemP informed me that the editor never seemed to actually work. Upon searching through the code, I found that several vars relating to sound environments were getting set, but never being used anywhere throughout the code.
This issue appears to have been around since retail 663b3471b9e14c31e7102af8b3f76c1aa465c932. What's the best way to go about correcting this?

**Comment by notimaginative** [comment]
Just tested this and sound environments appear to work fine for me in game. I don't think I've tested it in FRED since the feature was added however so I don't really know what the status is there. But when the editor was added to FRED I'm certain it worked at the time (wouldn't have added it otherwise).

To my knowledge, which is based on the public source code, which could have been changed by Volition before release...
Retail FS2 only had a single sound environment which was either enabled or disabled. Retail FS1 had two environments, one for general missions and one for the final subspace mission. Actual sound environment support (which could still be greatly improved) was added during the first sound code overhaul back in the 2009-2010 time frame.

**Comment by wookieejedi** [comment]
Agreed, sound environments themselves definitely work, tested on recent nightlies even. It works, but the selection box in FRED is like 1 line it looks like, so I have to use Arrow keys to set the sound environment (or manually add it to the mission file).

**Comment by MjnMixael** [comment]
Need to circle back and check on this for QtFRED after #6945 is merged. The dialog seems to work better than original FRED and sets what seem to be the proper variables.
